KBH/3Day/문자열 다루기 기본.js

```diff
@@ -0,0 +1,14 @@
+function solution(s) {
+    let answer = true;
+    let St = s.split("");
```

Comment:
```suggestion
    let St = s.split("");
```
@KimBYH  `St`와 같은 네이밍은 절대 피하셔야되요. 어떤 의미의 변수인지 전혀 파악이 안됩니다.
또한, 기본적으로 자바스크립트에서는 카멜케이스 네이밍 컨벤션을 갖습니다.
지금처럼 첫 글자가 대문자인 파스칼 케이스의 경우에는 클래스, 생성자 함수 등에서만 사용되요.
키워드 알려드렸으니 구글링해서 한번 확인해보세요 😄

혹시 좋은 코드란 무엇인가를 생각해보셨을까요? ㅎㅎ 한번 고민해보시고 답변해주세요

Reply:
또한, 항상 값의 재할당이 없다면 const로 변수를 정의하는게 좋습니다.
코드를 작성할 때 최우선적으로 const로 정의하고, 값의 재할당이 필요하다면 let으로 변경하는 습관이 안정적인 코드를 작성하는데 도움이 됩니다.

Reply:
`St = s.split("");`의 같은 경우에 다른 리뷰어들이 봐도 어떤 의미에서 사용한 변수인지 모를수도 있겠네요.  배열 나누기의 의미를 가진 `cosnt arraySplit`으로 사용해도 될까요?

Reply:
@KimBYH 
>문제 설명
문자열 s의 길이가 4 혹은 6이고, 숫자로만 구성돼있는지 확인해주는 함수, solution을 완성하세요. 예를 들어 s가 "a234"이면 False를 리턴하고 "1234"라면 True를 리턴하면 됩니다.

s라는 문자열을 `split('')`으로 개별 문자로 구성된 배열로 분할했기 때문에 `characters`가 적절해보입니다 어떻게 생각하실까요? 
- 단축어로 `chars` 도 있습니다. 
- 하지만 단축어가 매번 좋은건 아닙니다. 짧은 코드보다 길더라도 명확한 의미 전달이 훨씬 중요합니다.

배열의 네이밍은 `~~{Array}` 등도 있지만 복수형을 통해 나타내기도 합니다.
```ts
//ex. 숫자로 구성된 배열 numbers
const numbers = [1, 2, 3, 4, 5]
```

또한, 변수 네이밍은 `명사형`으로 표현한다고 생각하시면 되요.
예를 들어, "필터링 된 숫자 배열"은 `filteredNumbers` 를 예로 들 수 있습니다.
피드백 받은 내용들을 9기분들께 잘 공유해주세요 😄

Reply:
늦게 코멘트 달아서 죄송합니다!
각 인덱스 안에 문자값만 저장이 되는거기 때문에
char나 character등으로도 사용할 수 있는것을 처음 알았습니다! 다른 프로그래밍 언어에서는 char가 예약어로서 사용되는걸로 알고있어서 검색해봤는데 따로 char로 명시적 형변환이 없어서 변수명으로 선언할 수 있었네요.
또 말씀하신대로 char라고 하면 편하긴 하지만 그래도 명확하게 character로 선언하는것이 다른 리뷰어들이 보기에는 편하겠네요!